nca/Parsers/IngressPolicyYamlParser.py

```diff
@@ -242,10 +243,14 @@ def parse_policy(self):
 
         self.default_backend_peers, self.default_backend_ports = self.parse_backend(policy_spec.get('defaultBackend'),
                                                                                     True)
-        # TODO extend to other ingress controllers
```

Comment:
There are many more other ingress controllers, so I would keep this comment.

Reply:
done